plugins/Sf2Player/Sf2Player.cpp

```diff
@@ -627,12 +582,12 @@ void Sf2Instrument::reloadSynth()
 	{
 		// Now, delete the old one and replace
 		m_synthMutex.lock();
-		fluid_synth_remove_sfont( m_synth, m_font->fluidFont );
+		fluid_synth_remove_sfont( m_synth, m_font );
 		delete_fluid_synth( m_synth );
 
 		// New synth
 		m_synth = new_fluid_synth( m_settings );
-		m_fontId = fluid_synth_add_sfont( m_synth, m_font->fluidFont );
+		m_fontId = fluid_synth_add_sfont( m_synth, m_font );
```

Comment:
```suggestion
		m_fontId = fluid_synth_add_sfont(m_synth, m_font);
```

Reply:
I'll leave this one as-is for consistency.